audobject/core/utils.py

```diff
@@ -6,20 +6,53 @@
 import typing
 import warnings
 
+from importlib_metadata import packages_distributions
+
 import audeer
 
 from audobject.core.config import config
 from audobject.core import define
 
 
+def create_class_key(cls: type, include_version: bool) -> str:
+    r"""Create class key."""
```

Comment:
Could you maybe explain in the docstring what a class key is.

Reply:
![image](https://user-images.githubusercontent.com/10383417/158606544-5c5f6e03-2555-4b73-a1db-4d505a6daf21.png)

![image](https://user-images.githubusercontent.com/10383417/158606581-d59d4680-429c-4421-96ce-148cdca93866.png)

Reply:
Thanks.